modules/vectorfieldvisualizationgl/src/processors/2d/vector2dmagnitude.cpp

```diff
@@ -59,4 +59,13 @@ LayerConfig Vector2DMagnitude::outputConfig(const Layer& input) const {
                                       .valueRange = dvec2{0.0, conservativeMax}});
 }
 
+void Vector2DMagnitude::preProcess(TextureUnitContainer&, const Layer& input, Layer&) {
```

Comment:
I understand that the processor is "wrong", but I don't think this is the correct solution...
And I don't think this "fix" solves it for none float data.

Reply:
I don't think getNormalizedTexel is appropriate here. its main purpose for tf lookups. 
We should probably add some more functions for this

Also have to think about how the result is converted "back" to what ever format it used...
What if we write to a uint16 texture? What should we put in FragData0?